src/vs/workbench/contrib/customEditor/browser/webviewEditor.contribution.ts

```diff
@@ -46,21 +46,23 @@ Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration)
 		'properties': {
 			[customEditorsAssociationsKey]: {
 				type: 'array',
-				markdownDescription: nls.localize('editor.editorAssociations', "Configure which editor to use for a resource."),
+				markdownDescription: nls.localize('editor.editorAssociations', "Configure which editor to use for specific file types."),
 				items: {
 					type: 'object',
+					defaultSnippets: [{
+						body: {
+							'viewType': '$1',
+							'filenamePattern': '$2'
```

Comment:
You might want to look at the defaults for `files.associations`, which gives sample patterns for people who don't know what the glob patterns should look lik.e